src/SegmentedControl/SegmentedControl.less

```diff
@@ -0,0 +1,27 @@
+@import "../fonts.less";
+
+.SegmentedControl {
+  display: inline-block;
+  margin-right: 10px;
+  .segmented_control--selectable-item {
+    background-color: #FFFFFF;
+    color: #566279;
+    font-size: 16px;
+    font-family: "Proxima Nova";
+    font-weight: 600;
+    text-align: center;
+    display: inline-block;
+    padding: 10px 20px;
+    border-left: 1px solid #E3E5EB;
+    &.selected {
+      background-color: #4274F6;
+      color: #FAFAFC;
+    }
+    &:first-child {
+      border-left: none;
+    }
+  }
+  border: 1px solid #E3E6EB;
+  border-radius: 3px;
+}
```

Comment:
nit: a little tough to visually parse these rules.
i'd recommend:
- write all `.SegmentedControl` rules first before sub-component rulesets
- add an empty line before nested rulesets 

optional uber-nit: alpha for the rules in each block, since it's a lot easier to find a specific rule when scanning the list